okhttp-clients/src/test/java/com/palantir/conjure/java/okhttp/IoExceptionResponseHandlerTest.java

```diff
@@ -51,7 +51,7 @@ public void extractsIoExceptionForAllErrorCodes() {
         for (int code : ImmutableList.of(300, 400, 404, 500)) {
             IOException exception = decode(MediaType.APPLICATION_JSON, code, "body").get();
             assertThat(exception.getMessage()).isEqualTo(
```

Comment:
nit: could use `assertThatLoggableExceptionThrownBy` here, even though safe-logging wasn't introduced by your change.

Reply:
We're not throwing an exception here and the `decode` method returns a regular `IOException`. So we can either create a callable that throws this exception or cast it to a `SafeIoException` but both of these options seems a bit gross. Happy to do this if you would prefer though.